### ISSUE TITLE
Sending query params with multiple values for the same key

### ISSUE THREAD

**Issue by nebularazer** [klen/aioauth-client#119]
When using the google [people batch api](https://developers.google.com/people/api/rest/v1/people/getBatchGet). 

You have to construct an url like this.
`https://people.googleapis.com/v1/people:batchGet?resourceNames=people/12345&resourceNames=people/67890&personFields=photos`

This fails when aioauth-client tries to modify the params to add the access token:
```
File "~/.venv/lib/python3.6/site-packages/aioauth_client.py", line 315, in request
    params[self.access_token_key] = access_token
TypeError: list indices must be integers or slices, not str
```

**Comment by nebularazer** [comment]
I have to reopen my issue again, as my own fix turned out to cause problems.
My fix just always added the tuple `('access_token', '<token>')` to the params.
When reusing params will can end up with multiple `access_token=123&access_token=123` and at least googles people api does not like that -> 401
I to avoid that i added this:
``` python
params = [
    (key, value) for key, value in params
    if key != self.access_token_key
]
```

i will send another PR later...

**Comment by klen** [comment]
fixed since 0.16.2
